app/services/validator_check_active_service.rb

```diff
@@ -86,16 +86,7 @@ def update_scorable(validator)
 
   # returns true if validator was not delinquent for any period of time since DELINQUENT_TIME
   def not_delinquent?(validator)
-    nondelinquent_history = ValidatorHistory.where(
-                                              account: validator.account,
-                                              delinquent: false
-                                            )
-                                            .where(
-                                              'created_at > ?',
-                                              DateTime.now - @delinquent_time
-                                            )
-
-    nondelinquent_history.exists?
+    !validator.delinquent?
```

Comment:
I would merge the two options like `nondelinquent_history.exists? || !validator.delinquent?`